CONTRIBUTING.md

```diff
@@ -35,6 +35,24 @@ improve its performance, do not hesitate to fork and send pull requests.
    - Avoid `eval` if possible.
    - Prefer `zstyle` over environment variables for configuration.
 
+#### Using an Alternative zprezto Directory
+
+To work on zprezto without messing with your current configuration:
+
+    mkdir devel-zprezto
```

Comment:
Please use fenced code blocks here (the 3 back ticks). We had issues with these blocks rendering improperly when uses spaces to trigger a code block.